content/docs/app-developer-guide/export-to-oci-layout.md

```diff
@@ -0,0 +1,151 @@
++++
+title="Export to OCI layout format"
+weight=3
+summary="Learn how to export your application image to disk in OCI layout format"
++++
+
+<div class="quote mb-4">
+    The OCI Image Layout is the directory structure for OCI content-addressable blobs and location-addressable references.
+    <div class="author">See the <a href="https://github.com/opencontainers/image-spec/blob/main/image-layout.md">specification</a></div>
+</div>
+
+Exporting to OCI layout format is an **experimental** feature available on pack since version X.Y.Z
+
+### 1. Enable experimental feature
+
+Verify your pack version is equal or greater than X.Y.Z
+
+```bash
+pack version
+```
+
+Enable the experimental features on pack
+
+```bash
+pack config experimental true
+```
+
+You can confirm everything is fine, checking the `config.toml` file in your `PACK_HOME` installation folder, for example:
+
+```bash
+cat ~/.pack/config.toml
+experimental = true
+layout-repo-dir = "<$HOME>/.pack/layout-repo"
+```
+
+The configuration shows the experimental mode was **enabled** and a local directory to save images on disk was configured to path `<$HOME>/.pack/layout-repo`.
+
+### 2. Build the app
+
+If you haven't already, please follow the steps to [build an app](/docs/app-developer-guide/build-an-app), once you managed to build a sample application you can try exporting the sample application to disk in OCI layout format. 
+
+The OCI layout feature must be enabled using the convention `oci:<path/to/save/image>` in the `<image-name>` parameter when invoking `pack build`.
+
+For example:
+
+```bash
+pack build oci:sample-app --path samples/apps/java-maven --builder cnbs/sample-builder:bionic
+```
+
+It will save the image in a folder `./sample-app` created in your current directory.
```

Comment:
Why do we need `layout-repo-dir` if the OCI layout is saved to the PWD?

Reply:
We are using the `layout-repo-dir` to save the `run-image` on disk because actually invoking the lifecycle.  When lifecycle is configured to use OCI layout it is expecting to have all the require inputs (like the run-image) on disk,  because that should be transparent for users, we are saving those images in that particular folder.